Assets/MixedRealityToolkit.SDK/Features/UX/Prefabs/Slate/Slate.prefab

```diff
@@ -254,8 +253,13 @@ MonoBehaviour:
           m_StringArgument: 
           m_BoolArgument: 0
         m_CallState: 2
-    m_TypeName: UnityEngine.Events.UnityEvent, UnityEngine.CoreModule, Version=0.0.0.0,
-      Culture=neutral, PublicKeyToken=null
+    m_TypeName: Microsoft.MixedReality.Toolkit.Input.PanUnityEvent, Microsoft.MixedReality.Toolkit.SDK,
```

Comment:
Was this switch to PanUnityEvent something that was swapped automatically during the save? This seems like it would be a change that's unrelated to this radial solver update at first glance.

Reply:
It seems like it was automatically updated by Unity.